scripts/velocity.py

```diff
@@ -1,5 +1,25 @@
 """
 Generates reports on recently-closed issues on ZenHub and GitHub.
+
+To run this script locally, printing the report to stdout, the
+following must be configured in your environment.local.py:
+* azul_velocity_github_token
```

Comment:
Precedent in Azul is to have an empty line between paragraphs and in my opinion, bulleted items are paragraphs.